frontend/packages/dev-console/src/components/edit-deployment/__tests__/AdvancedSection.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { render, screen, fireEvent, cleanup, waitFor } from '@testing-library/react';
+import { Resources } from '../../import/import-types';
+import MockForm from '../__mocks__/MockForm';
+import AdvancedSection from '../AdvancedSection';
+
+window.HTMLElement.prototype.scrollIntoView = () => {}; // scrollIntoView is not implemented in jsdom
+
+const handleSubmit = jest.fn();
+
+beforeEach(() =>
+  render(
+    <MockForm handleSubmit={handleSubmit}>
+      {() => <AdvancedSection resourceType={Resources.OpenShift} />}
+    </MockForm>,
+  ),
+);
+
+afterEach(() => cleanup());
+
+test('AdvancedSection: should show Pause rollouts section on click', async () => {
```

Comment:
I think we can still use `describe` and `it` blocks like we do with all of our tests. This is the note that I found on official RTL docs - 
> Note: you can also use describe and it blocks with React Testing Library. React Testing Library doesn't replace Jest, just Enzyme. We recommend test because it helps with this: [Avoid Nesting When You're Testing](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing)